app/test/process_proposal_test.go

```diff
@@ -216,3 +279,11 @@ func flipSequenceStart(share shares.Share) shares.Share {
 	share[infoByteIndex] = share[infoByteIndex] ^ 0x01
 	return share
 }
+
+func deref[T any](s []*T) []T {
```

Comment:
go really needs simple generic funcs like these in the standard lib or something